pysonde/pysonde.py

```diff
@@ -153,6 +161,23 @@ def main(args=None):
                 snd.convert_sounding_df2ds()
                 snd.create_dataset(cfg)
                 snd.export(args["output"], cfg)
+
+        elif isinstance(reader, readers.readers.METEOMODEM):
+            sounding_asc, sounding_dsc = sounding.split_by_direction()
+            for snd in [sounding_asc, sounding_dsc]:
+                if len(snd.profile) < 2:
+                    logging.warning(
+                        "Sounding ({}) does not contain data. "
+                        "Skip sounding-direction of {}".format(
+                            snd.meta_data["sounding_direction"], file
+                        )
+                    )
+                    continue
+                snd.calculate_additional_variables(cfg)
+                snd.convert_sounding_df2ds()
+                snd.create_dataset(cfg)
+                snd.export(args["output"], cfg)
+
```

Comment:
This is basically the same as for the MWX reader so I just adjusted the is instance statement to reduce duplication of code.